config/deploy.rb

```diff
@@ -86,10 +86,17 @@
   desc 'Modify permissions on /srv/idp'
   task :mod_perms do
     on roles(:web), in: :parallel do
-      execute :sudo, :chown, '-R', 'ubuntu:nogroup', deploy_to
+      execute :sudo, :chown, '-R', 'ubuntu:nogroup', deploy_to + '/current', '/srv/idp/shared'
```

Comment:
In my experience the `chown` has occasionally timed out (too many files?) so adding more directories and files to this seems like it would make things worse? And potentially block things after it like `passenger:restart` ?